hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -299,11 +299,30 @@
       tests: []
     }
 
-    //////////////////////////////////////////////////////////////////////////////
-    // System Peripherals                                                       //
-    // RV_DM, RV_TIMER, AON_TIMER, PLIC, CLK/RST/PWR MGR, ALERT_HANDLER, LC_MGR //
-    //////////////////////////////////////////////////////////////////////////////
+    ////////////////////////////////////////////////////////////////////////////////////
+    // System Peripherals                                                             //
+    // XBAR, RV_DM, RV_TIMER, AON_TIMER, PLIC, CLK/RST/PWR MGR, ALERT_HANDLER, LC_MGR //
+    ////////////////////////////////////////////////////////////////////////////////////
 
+    // XBAR (pre-verified IP) tests:
+    {
+      name: chip_data_integrity
+      desc: '''
+            Verify the alert signaling mechanism due to data integrity violation.
+
+            An automated SW test which performs the following for each IP:
+            - Pick a CSR to write.
+            - Corrupts the control / data / integrity bits at the CPU source via force.
+            - Verify that the device detects the integrity violation causing an alert.
+            - Verifying the alert upto the NMI stage.
+            - Now pick a CSR to read.
+            - Corrupt the control / data / integrity bits at the destination (IP) via force.
+            - Verify that the CPU detects the integrity violation causing an alert.
+            - Verify the alert upto the NMI stage.
+            '''
+      milestone: V2
+      tests: []
+    }
```

Comment:
Do we need to test corrupting bits over multiple clocks, or is that done solely at the IP level?   
@tjaychen @moidx do we need to capture error log information (ie src, dst, syndrome, etc)